jinahub/indexers/searcher/compound/FaissPostgresSearcher/tests/integration/test_208.py

```diff
@@ -0,0 +1,25 @@
+import os
```

Comment:
Not obvious why `test_208.py`!? I want to know the background story! Some smear campaign involved?

Reply:
> Also fixes #208